app/src/main/java/cm/aptoide/pt/install/installer/DefaultInstaller.java

```diff
@@ -396,13 +396,8 @@ private Installation updateInstallation(Installation installation, int type, int
               updateInstallation(installation,
                   shouldSetPackageInstaller ? RoomInstalled.TYPE_PACKAGE_INSTALLER
                       : RoomInstalled.TYPE_DEFAULT, map(installStatus));
-              if (installStatus.getStatus()
-                  .equals(InstallStatus.Status.FAIL) && isDeviceMIUI()) {
-                installerAnalytics.sendMiuiInstallResultEvent(InstallStatus.Status.FAIL);
+              if (installStatus.getStatus().equals(InstallStatus.Status.FAIL)) {
```

Comment:
InstallStatus.Status.FAIL only gets set as the status in 4 places. 3 of them are situations where the activity somehow wound being null, which is super-unlikely. The fourth is if there was a conflict when attempting to install the .apk. By calling startInstallIntent, we'll get the conflict a second time but this tile Aptoide will show an error screen informing the user of the conflict.

Nothing automatic will occur as far as uninstalling or reinstalling anything, but at least they'll know something's happened & that they need to do something manually.

Reply:
Nirmal reviewed this PR & approved it (he doesn't have GitHub access at the moment).